apps/google-analytics-4/frontend/src/index.tsx

```diff
@@ -6,6 +8,19 @@ import { SDKProvider } from '@contentful/react-apps-toolkit';
 
 import LocalhostWarning from './components/LocalhostWarning';
 import App from './App';
+import { config } from './config';
+
+Sentry.init({
+  dsn: config.sentryDSN,
+  integrations: [new BrowserTracing()],
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production
+  tracesSampleRate: 1.0,
+  environment: process.env.NODE_ENV,
+  // TODO: setup Sentry as part of release pipeline (see: https://docs.sentry.io/platforms/javascript/sourcemaps/?_ga=2.56533545.342806665.1676988870-873194326.1675171780#uploading-source-maps-to-sentry)
```

Comment:
ticket filed for this. will remove this TODO when it's been completed

Reply:
@mgoudy91 might be a good one for you to pick up for us while you're in the build stuff https://contentful.atlassian.net/browse/INTEG-171